src/components/MainScene.jsx

```diff
@@ -67,17 +88,29 @@ export default class MainScene extends Component {
     animate = () => {
         this.frameId = window.requestAnimationFrame(this.animate);
         this.renderer.render(this.scene, this.camera);
+        this.helperAnimate();
         this.controls.update();
+    };
 
+    helperAnimate = () => {
+        this.helperCamera.position.copy(this.camera.position);
+        this.helperCamera.position.sub(this.controls.target);
+        this.helperCamera.position.setLength(300);
+        this.helperCamera.lookAt(this.helperScene.position);
+        this.helperRenderer.render(this.helperScene, this.helperCamera);
     };
 
     render() {
-        return (<Fragment>
+        return (<div id="mainScene">
                 <div id="boardCanvas" ref={mount => {
                     this.mount = mount
+                }}>
+                </div>
+                <div id="helperCanvas" ref={helper => {
```

Comment:
Could you update the inline ref callback function? It creates a brand new function for every render. Therefore, Inline function binding causing the bad performance impact.